### PR TITLE
t/95: Remove the Controller class

### LINKED ISSUE #95: Remove the Controller class
All changes required to remove the `Controller` class.
1. `View#createCollection()`.
2. Add `ViewCollection()` (if needed).
3. Make that `ViewCollection` instance can be used directly in a template and that `view.someCollection.add( someView )`  or `remove( someView )`.
4. Verify `init()` and `destroy()` chains.

### PR DESCRIPTION
**Pull request by oleq**
Closes #95.

### CODE REVIEW

File: src/view.js [hunk 1]
```diff
@@ -22,10 +22,14 @@ export default class View {
 	/**
 	 * Creates an instance of the {@link ui.View} class.
 	 *
+	 * TODO: A simple example how to create one.
+	 *
 	 * @param {utils.Locale} [locale] The {@link core.editor.Editor#locale editor's locale} instance.
 	 */
 	constructor( locale ) {
```
Comment (pjasiun) [inline src/view.js]: I remember we talked about View taking `attributes ` object as a second parameter, so one can easily initialize multiple observable creating a view class. I'm not sure do we need to make it generic, so should such parameter be in the base `View` class or in specific views, if needed.

**Comment by Reinmar** [comment]
The PRs to review:

- [ ] this one 
- [ ] https://github.com/ckeditor/ckeditor5-ui-default/pull/104
- [ ] https://github.com/ckeditor/ckeditor5-theme-lark/pull/39
- [ ] https://github.com/ckeditor/ckeditor5-editor-classic/pull/35
- [ ] https://github.com/ckeditor/ckeditor5-core/pull/39
- [ ] https://github.com/ckeditor/ckeditor5-list/pull/21
- [ ] https://github.com/ckeditor/ckeditor5-basic-styles/pull/32
- [ ] https://github.com/ckeditor/ckeditor5-heading/pull/42
- [ ] https://github.com/ckeditor/ckeditor5-undo/pull/46
- [ ] https://github.com/ckeditor/ckeditor5-link/pull/58

**Comment by Reinmar** [comment]
I decided to merge all this code to master and review it there, because we risk having too many conflicts too quickly and blocking ourselves. 3 of you worked on these changes, all tests pass, CC is 100%, so let's not keep the world (:D) waiting.

**Comment by Reinmar** [comment]
I finished the review. I haven't found many things, which is great taken the size of the refactoring. I wrote down everything in https://github.com/ckeditor/ckeditor5-ui/issues/103.

Kudos guys!

**Pull request merged**